addons/live-comments/profiles.js

```diff
@@ -0,0 +1,139 @@
+    newDiv.innerHTML = newComments;
+
+    /// format reply for logged in
+    newDiv.querySelectorAll("a.reply").forEach((div) => {
+      div.style.display = "inline";
+      div.setAttribute("data-control", "reply-to");
+    });
+    newDiv.querySelectorAll(".truncated").forEach((div) => {
+      div.classList.remove("truncated");
+    });
+
+    // console.log(newDiv.children)
+
+    for (let tlr of newDiv.children) {
+      if (!tlr.classList.contains("top-level-reply")) {
+        continue;
+      }
+      // console.log(tlr)
+
+      let comment = tlr.firstElementChild;
+      let commentId = comment.id;
+      let alreadyOnPage = document.getElementById(commentId);
+      // console.log(alreadyOnPage)
+
+      if (alreadyOnPage) {
+        // loop through replies
+        let repliesElem = tlr.querySelector(".replies");
+        let replies = repliesElem.children;
+        for (let rli of replies) {
+          let rcomment = rli.firstElementChild;
+          let rcommentId = rcomment.id;
+          let ralreadyOnPage = document.getElementById(rcommentId);
+
+          if (!ralreadyOnPage) {
+            // console.log("NEW REPLY!!!");
+            let rolderSiblingId = rli.previousElementSibling?.firstElementChild?.id;
+            // console.log("sib", rolderSiblingId);
+            let rolderSiblingAlreadyOnPage = document.getElementById(rolderSiblingId)?.parentElement;
+            // console.log("rolder", rolderSiblingAlreadyOnPage);
+            rli.classList.remove("truncated"); // having truncated hides the comment element
+            rli.classList.add("newCom");
+            $(rli.querySelector("span.time"))?.timeago();
+            if (rolderSiblingAlreadyOnPage) {
+              rolderSiblingAlreadyOnPage.after(rli);
+              rolderSiblingAlreadyOnPage.classList.remove("last");
+            } else {
+              alreadyOnPage.parentElement.querySelector(".replies").appendChild(rli);
+            }
+          }
+        }
+      } else {
+        // console.log("SOMETHING NEW!!!");
+        // get older sibling id, and append new comment after that sibling on the document
+        let olderSiblingId = tlr.previousElementSibling?.firstElementChild?.id;
+        tlr.classList.add("newCom");
+        $(tlr.querySelector("span.time")).timeago();
+        let olderSiblingAlreadyOnPage = document.getElementById(olderSiblingId)?.parentElement;
+        if (olderSiblingAlreadyOnPage) {
+          olderSiblingAlreadyOnPage.after(tlr);
+          // console.log("AFTER!");
+        } else if (!olderSiblingAlreadyOnPage && !olderSiblingId) {
+          oldComments.firstElementChild.before(tlr);
+          // console.log("AT TOP!");
+        } else if (!olderSiblingAlreadyOnPage && olderSiblingId) {
+          oldComments.lastElementChild.after(tlr);
+          // console.log("AT BOTTOM!");
+        }
+      }
+
+      /// TODO: populate replies
+
+      // console.log(comment)
+    }
+  }
+
+  setInterval(refreshComments, 3000);
```

Comment:
I think this should be at least 10 seconds.
```suggestion
  setInterval(refreshComments, 10000);
```